server/middleware/token.js

```diff
@@ -32,6 +32,9 @@ function rewriteUserLiteral(req, currentUserLiteral) {
       debug('req.url has been rewritten from %s to %s', urlBeforeRewrite,
         req.url);
     }
+  } else if (!req.accessToken && currentUserLiteral &&
```

Comment:
http://loopback.io/doc/en/contrib/style-guide.html#indentation-of-multi-line-expressions-in-if

I am proposing the following:

```js
function rewriteUserLiteral(req, currentUserLiteral) {
  if (!currentUserLiteral) return;
  
  if (req.accessToken && req.accessToken.userId) {
    // ...
  } else if (...) {
    // ...
  }
}
```